src/renderer/views/AppSettings/AppSettings.tsx

```diff
@@ -1,6 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './AppSettings.scss';
+import { useDispatch, useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import { Button, MenuItem, Select } from '@mui/material';
+import {
+  selectCurrentLocale,
+  setLocale,
+} from '../../../shared/redux/slices/settingsSlice';
 
-const AppSettings = () => <div>AppSettings</div>;
+const AppSettings = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const [language, setLanguage] = useState(useSelector(selectCurrentLocale));
+
+  function submitChanges() {
+    dispatch(setLocale(language));
+  }
+
+  return (
+    <div>
+      {t('AppSettings.title')}
+      <Select
+        label={t('AppSettings.language')}
+        value={language}
+        onChange={(e) => setLanguage(e.target.value as 'en' | 'pl')}
```

Comment:
Just to future proof this a little (in case more languages come in the future) you could change the typing to `as typeof language` so that you don't have to come in and add them manually when new locales are added.

Reply:
done